pkgs/by-name/fi/firezone-server/package.nix

```
@@ -0,0 +1,90 @@
+{
+  lib,
+  nixosTests,
+  fetchFromGitHub,
+  beamPackages,
+  gitMinimal,
+  pnpm_9,
+  nodejs,
+  tailwindcss_3,
+  esbuild,
+
+  mixReleaseName ? "domain", # "domain" "web" or "api"
+}:
+
+beamPackages.mixRelease rec {
+  pname = "firezone-server-${mixReleaseName}";
+  version = "0-unstable-2025-03-15";
+
+  src = "${
+    fetchFromGitHub {
+      owner = "firezone";
+      repo = "firezone";
+      rev = "09fb5f927410503b0d6e7fc6cf6a2ba06cb5a281";
+      hash = "sha256-1CZBFhOwX0DfXykPQ9tzn4tHg2tSnByXEPtlZleHK5k=";
+
+      # This is necessary to allow sending mails via SMTP, as the default
+      # SMTP adapter is current broken: https://github.com/swoosh/swoosh/issues/785
+      postFetch = ''
+        ${lib.getExe gitMinimal} -C $out apply ${./0000-add-mua.patch}
+      '';
+    }
+  }/elixir";
+
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit pname version;
+    src = "${src}/apps/web/assets";
+    hash = "sha256-ejyBppFtKeyVhAWmssglbpLleOnbw9d4B+iM5Vtx47A=";
+  };
+  pnpmRoot = "apps/web/assets";
+
+  preBuild = ''
+    cat >> config/config.exs <<EOF
+    config :tailwind, path: "${lib.getExe tailwindcss_3}"
+    config :esbuild, path: "${lib.getExe esbuild}"
+    EOF
+
+    cat >> config/runtime.exs <<EOF
+    config :tzdata, :data_dir, System.get_env("TZDATA_DIR")
```

```suggestion
    config :tzdata, :data_dir, System.fetch_env!("TZDATA_DIR")
```

This will fail if it's unset, rather than silently setting it to nil.